browser/ui/views/tabs/tab_drag_controller.cc

```diff
@@ -327,6 +330,39 @@ gfx::Rect TabDragController::CalculateDraggedBrowserBounds(
   return bounds;
 }
 
+[[nodiscard]] TabDragController::Liveness TabDragController::ContinueDragging(
+    const gfx::Point& point_in_screen) {
+  if (!base::FeatureList::IsEnabled(tabs::features::kBraveSplitView)) {
+    return TabDragControllerChromium::ContinueDragging(point_in_screen);
+  }
+
+  const auto liveness =
+      TabDragControllerChromium::ContinueDragging(point_in_screen);
+
+  if (!attached_context_) {
+    // This is when drag session ends.
+    on_tab_drag_ended_closure_.RunAndReset();
+    return liveness;
+  }
+
+  auto* browser_widget = GetAttachedBrowserWidget();
```

Comment:
When dragging a tab from tile and drop to the outside of the original window,
got crashed here. maybe `browser_widget` is invalid at that time.

Reply:
Was it windows? On my Mac, I could reproduce it.

Reply:
yes, tested on Win11.

Reply:
Thanks! Added check.